route_following_plugin/src/route_following_plugin.cpp

```diff
@@ -207,7 +207,7 @@ namespace route_following_plugin
 
         for (auto &maneuver : maneuvers)
         {
-            time_progress += maneuverDuration(maneuver);
+            time_progress += getManeuverDuration(maneuver, 0.001);
```

Comment:
Here the epsilon is 0.001 while on line 167 it is 0.0001 what is the reason for the discrepancy? Seems like it should be a constant